core/tasks/handler/contact_tasks.go

```diff
@@ -674,6 +680,12 @@ type StopEvent struct {
 	OccurredOn time.Time        `json:"occurred_on"`
 }
 
+type MsgDeletedEvent struct {
+	OrgID     models.OrgID     `json:"org_id"`
+	ChannelID models.ChannelID `json:"channel_id"`
```

Comment:
channelID used anywhere? should we filter by that in the SQL too?

Reply:
I was thinking it might be worth recording to analytics but I might remove it